<ui version="4.0" >
 <class>DataViewer</class>
 <widget class="QMainWindow" name="DataViewer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Data Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>37</y>
     <width>976</width>
     <height>570</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible" >
       <bool>false</bool>
      </property>
      <widget class="QSplitter" name="splitterBlob" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="childrenCollapsible" >
        <bool>false</bool>
       </property>
       <widget class="QTabWidget" name="tabWidget" >
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>805</width>
           <height>266</height>
          </rect>
         </property>
         <attribute name="title" >
          <string>Full View</string>
         </attribute>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QTableView" name="tableView" >
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="alternatingRowColors" >
             <bool>true</bool>
            </property>
            <property name="selectionMode" >
             <enum>QAbstractItemView::ContiguousSelection</enum>
            </property>
            <property name="verticalScrollMode" >
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode" >
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>840</width>
           <height>420</height>
          </rect>
         </property>
         <attribute name="title" >
          <string>Item View</string>
         </attribute>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="0" >
           <widget class="SqlItemView" native="1" name="itemView" />
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Item View is read only now.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>840</width>
           <height>420</height>
          </rect>
         </property>
         <attribute name="title" >
          <string>Script Output</string>
         </attribute>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="SqlEditorWidget" name="scriptEdit" >
            <property name="readOnly" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="blobPreviewBox" >
        <property name="title" >
         <string>BLOB Preview</string>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="BlobPreviewWidget" native="1" name="blobPreview" />
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTextEdit" name="statusText" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>37</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Open a new standalone window with current results</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Row" />
   <addaction name="actionRemove_Row" />
   <addaction name="actionTruncate_Table" />
   <addaction name="separator" />
   <addaction name="actionCommit" />
   <addaction name="actionRollback" />
  </widget>
  <widget class="QToolBar" name="exportToolBar" >
   <property name="geometry" >
    <rect>
     <x>551</x>
     <y>0</y>
     <width>262</width>
     <height>37</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_2</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionBLOB_Preview" />
   <addaction name="actionExport_Data" />
   <addaction name="actionClose" />
  </widget>
  <widget class="QToolBar" name="snapshotToolBar" >
   <property name="geometry" >
    <rect>
     <x>813</x>
     <y>0</y>
     <width>163</width>
     <height>37</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_3</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionRipOut" />
  </widget>
  <action name="actionNew_Row" >
   <property name="text" >
    <string>New Row</string>
   </property>
  </action>
  <action name="actionRemove_Row" >
   <property name="text" >
    <string>Remove Row</string>
   </property>
  </action>
  <action name="actionTruncate_Table" >
   <property name="text" >
    <string>Truncate Table</string>
   </property>
  </action>
  <action name="actionExport_Data" >
   <property name="text" >
    <string>Export Data</string>
   </property>
  </action>
  <action name="actionCommit" >
   <property name="text" >
    <string>Commit</string>
   </property>
   <property name="toolTip" >
    <string>Commit the current transaction in the result table</string>
   </property>
  </action>
  <action name="actionRollback" >
   <property name="text" >
    <string>Rollback</string>
   </property>
   <property name="toolTip" >
    <string>Rollback the current transaction in the result table</string>
   </property>
  </action>
  <action name="actionRipOut" >
   <property name="text" >
    <string>Table Snapshot</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="text" >
    <string>Close</string>
   </property>
   <property name="visible" >
    <bool>false</bool>
   </property>
  </action>
  <action name="actionBLOB_Preview" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>BLOB Preview</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SqlEditorWidget</class>
   <extends>QTextEdit</extends>
   <header>sqleditorwidget.h</header>
  </customwidget>
  <customwidget>
   <class>SqlItemView</class>
   <extends>QWidget</extends>
   <header>sqlitemview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlobPreviewWidget</class>
   <extends>QWidget</extends>
   <header>blobpreviewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons/icons.qrc" />
 </resources>
 <connections/>
</ui>
